<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTypon 2017</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mytypon.qrc">
    <normaloff>:/pictures/chip.png</normaloff>:/pictures/chip.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuWorkspaces">
     <property name="title">
      <string>Workspaces</string>
     </property>
     <addaction name="actionLoadDefaultWSP"/>
     <addaction name="actionLoadWSPFromFile"/>
     <addaction name="actionSaveWSP"/>
     <addaction name="actionSaveWSPAs"/>
     <addaction name="actionSetCurrentWSPAsDefault"/>
     <addaction name="actionRestoreFactoryWSP"/>
    </widget>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionCloseFile"/>
    <addaction name="separator"/>
    <addaction name="menuWorkspaces"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionGeneralOptions"/>
    <addaction name="actionCurrentTyponOptions"/>
    <addaction name="separator"/>
    <addaction name="actionGrid"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actiontest"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <widget class="QMenu" name="menutools">
    <property name="title">
     <string>tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSelection">
    <property name="title">
     <string>Selection</string>
    </property>
    <addaction name="actionSelectAll"/>
    <addaction name="actionClearSelection"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_left"/>
    <addaction name="actionRotate_Right"/>
    <addaction name="actionRotate_angle"/>
    <addaction name="separator"/>
    <addaction name="actionHorizontal_mirror"/>
    <addaction name="actionVertical_mirror"/>
    <addaction name="separator"/>
    <addaction name="actionGroup"/>
    <addaction name="actionUngroup"/>
    <addaction name="separator"/>
    <addaction name="actionGenerate_component"/>
    <addaction name="separator"/>
    <addaction name="actionAlign_on_grid"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSelection"/>
   <addaction name="menutools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuWindows"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="filesToolBar">
   <property name="windowTitle">
    <string>Files</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionOpenFile"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="optionsToolBar">
   <property name="windowTitle">
    <string>Options</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGeneralOptions"/>
   <addaction name="actionGrid"/>
   <addaction name="actiontest"/>
   <addaction name="actiontest_2"/>
  </widget>
  <widget class="MYDockWidget" name="toolsDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="MYToolBox" name="toolBoxWidget"/>
  </widget>
  <widget class="MYDockWidget" name="layersDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Layers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="LayersWidget" name="layersWidget"/>
  </widget>
  <widget class="MYDockWidget" name="undoDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Undo/Redo</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="UndoWidget" name="undoWidget"/>
  </widget>
  <widget class="MYDockWidget" name="miniViewDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Mini view</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="MiniView" name="miniViewWidget"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>Edit</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
  </widget>
  <widget class="QToolBar" name="selectionToolBar">
   <property name="windowTitle">
    <string>Selection</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionRotate_left"/>
   <addaction name="actionRotate_Right"/>
   <addaction name="actionRotate_angle"/>
   <addaction name="separator"/>
   <addaction name="actionHorizontal_mirror"/>
   <addaction name="actionVertical_mirror"/>
   <addaction name="separator"/>
   <addaction name="actionGroup"/>
   <addaction name="actionUngroup"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionAlign_on_grid"/>
   <addaction name="separator"/>
   <addaction name="actionGenerate_component"/>
  </widget>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/document_plain_new.png</normaloff>:/pictures/document_plain_new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/document_find.png</normaloff>:/pictures/document_find.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/document_delete.png</normaloff>:/pictures/document_delete.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRecent_files">
   <property name="text">
    <string>Recent files</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/disk_blue.png</normaloff>:/pictures/disk_blue.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/disks.png</normaloff>:/pictures/disks.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionLoadDefaultWSP">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/workspace_default.png</normaloff>:/pictures/workspace_default.png</iconset>
   </property>
   <property name="text">
    <string>Load default</string>
   </property>
  </action>
  <action name="actionLoadWSPFromFile">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/workspace_open.png</normaloff>:/pictures/workspace_open.png</iconset>
   </property>
   <property name="text">
    <string>Load from file...</string>
   </property>
  </action>
  <action name="actionSaveWSP">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/workspace_save.png</normaloff>:/pictures/workspace_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSetCurrentWSPAsDefault">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/Workspace_setdefault.png</normaloff>:/pictures/Workspace_setdefault.png</iconset>
   </property>
   <property name="text">
    <string>Set current as default</string>
   </property>
  </action>
  <action name="actionSaveWSPAs">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/workspace_saveas.png</normaloff>:/pictures/workspace_saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionRestoreFactoryWSP">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/workspace_factory.png</normaloff>:/pictures/workspace_factory.png</iconset>
   </property>
   <property name="text">
    <string>Restore factory settings</string>
   </property>
  </action>
  <action name="actionCurrentTyponOptions">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/page_gear.png</normaloff>:/pictures/page_gear.png</iconset>
   </property>
   <property name="text">
    <string>Current typon options...</string>
   </property>
  </action>
  <action name="actionGeneralOptions">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/gear.png</normaloff>:/pictures/gear.png</iconset>
   </property>
   <property name="text">
    <string>General options...</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/table.png</normaloff>:/pictures/table.png</iconset>
   </property>
   <property name="text">
    <string>Grid...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/help2.png</normaloff>:/pictures/help2.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/about.png</normaloff>:/pictures/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/page_copy.png</normaloff>:/pictures/page_copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/page_white_paste.png</normaloff>:/pictures/page_white_paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/cut.png</normaloff>:/pictures/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/zoom_in.png</normaloff>:/pictures/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/zoom_out.png</normaloff>:/pictures/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionToolbars_2">
   <property name="text">
    <string>Toolbars</string>
   </property>
  </action>
  <action name="actionTools_2">
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionRotate_left">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/shape_rotate_anticlockwise.png</normaloff>:/pictures/shape_rotate_anticlockwise.png</iconset>
   </property>
   <property name="text">
    <string>Rotate left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRotate_Right">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/shape_rotate_clockwise.png</normaloff>:/pictures/shape_rotate_clockwise.png</iconset>
   </property>
   <property name="text">
    <string>Rotate Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionRotate_angle">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/transform_rotate.png</normaloff>:/pictures/transform_rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate angle...</string>
   </property>
  </action>
  <action name="actionHorizontal_mirror">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/shape_flip_horizontal.png</normaloff>:/pictures/shape_flip_horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Horizontal mirror</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionVertical_mirror">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/shape_flip_vertical.png</normaloff>:/pictures/shape_flip_vertical.png</iconset>
   </property>
   <property name="text">
    <string>Vertical mirror</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/shape_group.png</normaloff>:/pictures/shape_group.png</iconset>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionUngroup">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/shape_ungroup.png</normaloff>:/pictures/shape_ungroup.png</iconset>
   </property>
   <property name="text">
    <string>Ungroup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="actionGenerate_component">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/new_component.png</normaloff>:/pictures/new_component.png</iconset>
   </property>
   <property name="text">
    <string>Generate component...</string>
   </property>
  </action>
  <action name="actionSend_to_Layer">
   <property name="text">
    <string>Send to Layer</string>
   </property>
  </action>
  <action name="actionAlign_on_grid">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/align.png</normaloff>:/pictures/align.png</iconset>
   </property>
   <property name="text">
    <string>Align on grid</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionClearSelection">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="mytypon.qrc">
     <normaloff>:/pictures/cross.png</normaloff>:/pictures/cross.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actiontest_2">
   <property name="text">
    <string>test2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MYDockWidget</class>
   <extends>QDockWidget</extends>
   <header>mydockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayersWidget</class>
   <extends>QWidget</extends>
   <header>layers/layerswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UndoWidget</class>
   <extends>QWidget</extends>
   <header>undowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MiniView</class>
   <extends>QWidget</extends>
   <header>miniview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MYToolBox</class>
   <extends>QWidget</extends>
   <header>toolswidget\mytoolbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mytypon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
